spec/services/broadcasts/welcome_notification/generator_spec.rb

```diff
@@ -6,8 +6,10 @@
   let!(:welcome_broadcast)         { create(:welcome_broadcast) }
   let!(:twitter_connect_broadcast) { create(:twitter_connect_broadcast) }
   let!(:github_connect_broadcast)  { create(:github_connect_broadcast) }
+  let!(:customize_broadcast)       { create(:customize_broadcast) }
 
   before do
+    allow(Notification).to receive(:send_welcome_notification).and_call_original
```

Comment:
What does `.and_call_original` do?

Reply:
@juliannatetreault The combination of `allow(SomeClass).to receive(:some_method)` and `and_call_original` allows you to call a method another object and call the actual method on that object.

In this example, we're saying that we want to allow ourselves to call `send_welcome_notification` on the class `Notification`, and when we do, we want to call the actual original method `send_welcome_notification`, which is defined on that class. This is a pattern used for mocking the response from another object (in this case, another class, the `Notification` class), when we don't want to actually _test_ that external object, but we do want to be able to receive the output of calling a method (`send_welcome_notification`) on that class.

You can read more about how this works in the documentation [here](https://relishapp.com/rspec/rspec-mocks/docs/configuring-responses/calling-the-original-implementation) and [here](https://relishapp.com/rspec/rspec-mocks/v/2-14/docs/message-expectations/calling-the-original-method).